public/app/features/dashboard-scene/saving/provisioned/PreviewBannerViewPR.tsx

```diff
@@ -0,0 +1,51 @@
+import { textUtil } from '@grafana/data';
+import { Trans, t } from '@grafana/i18n';
+import { Alert, Icon, Stack } from '@grafana/ui';
+
+import { commonAlertProps } from './DashboardPreviewBanner';
+
+// TODO: We have this https://github.com/grafana/git-ui-sync-project/issues/166 to add more details about the PR.
+
+/**
+ * @description This component is used to display a banner when a provisioned dashboard/folder is created or loaded from a new branch in Github.
+ */
+export function PreviewBannerViewPR({
+  prParam,
+  isFolder = false,
+  isNewPr,
+}: {
+  prParam: string;
+  isFolder?: boolean;
+  isNewPr?: boolean;
+}) {
+  const newPrText = isFolder
+    ? 'A new folder has been created in a branch in GitHub.'
+    : 'This dashboard is loaded from a branch in GitHub.';
+
+  const openPrText = isFolder
+    ? 'A new folder has been created in a pull request in GitHub.'
+    : 'This dashboard is loaded from a pull request in GitHub.';
```

Comment:
I don't think this will work, the translations need to be static. You'll need to do smth like 
```tsx
const openPrText = isFolder
    ? t('provisioned-dashboard-preview-banner.title-dashboard-loaded-branch-git-hub', 'A new folder has been created in a pull request in GitHub.')
    : t('provisioned-folder-preview-banner.title-dashboard-loaded-branch-git-hub', 'This dashboard is loaded from a pull request in GitHub.');
```